star_ratings/models.py

```diff
@@ -9,7 +9,7 @@
 from django.contrib.contenttypes.fields import GenericForeignKey
 from django.contrib.contenttypes.models import ContentType
 from django.db.models import Avg, Count, Sum
-from django.utils.encoding import python_2_unicode_compatible
+from six import python_2_unicode_compatible
```

Comment:
we dont need this since we've dropped py2

Reply:
Yeah, I figured it might not hurt to leave it in case people really dig their heels in, but there's every likelihood they'll have other problems in that case.